.config/tsaconfig.json

```
@@ -0,0 +1,10 @@
+{
+    "instanceUrl": "https://devdiv.visualstudio.com/",
+    "template": "TFSDEVDIV",
+    "projectName": "DEVDIV",
+    "areaPath": "DevDiv\\NET Fundamentals\\.NET Acquisition\\Docker",
+    "iterationPath": "DevDiv",
+    "notificationAliases": [ "dotnetADTeam@microsoft.com" ],
+    "repositoryName": "dotnet-buildtools-prereqs-docker",
```

This is the actual repo name in AzDO:

```suggestion
    "repositoryName": "dotnet-dotnet-buildtools-prereqs-docker",
```

---

The repo name don't have to match AzDO from other config files I have seen ([example](https://github.com/dotnet/arcade/blob/main/.config/tsaoptions.json#L8)).  The documentation doesn't call out how this is used.

---

Given the majority of repos are using the GH repo name, I decided to use it versus the AzDO repo name.